imqsauth/http.go

```diff
@@ -226,6 +226,7 @@ func (x *ImqsCentral) RunHttp() error {
 	smux.HandleFunc("/userobjects", x.makeHandler(HttpMethodGet, httpHandlerGetUsers, handlerFlagNeedAdminRights))
 	smux.HandleFunc("/groups", x.makeHandler(HttpMethodGet, httpHandlerGetGroups, 0))
 	smux.HandleFunc("/hasactivedirectory", x.makeHandler(HttpMethodGet, httpHandlerHasActiveDirectory, 0))
+	smux.HandleFunc("/groupspermnames", x.makeHandler(HttpMethodGet, httpHandlerGetGroupsPermNames, handlerFlagNeedAdminRights))
```

Comment:
I think this API names should have some underscores in it, because as it currently stands, it's unreadable.

Reply:
Done